packages/react/package.json

```diff
@@ -1042,6 +1042,370 @@
+      "types": "./lib-commonjs/Persona.d.ts",
+      "require": "./lib-commonjs/Persona.js"
+    },
+    "./lib-commonjs/PersonaCoin": {
+      "types": "./lib-commonjs/PersonaCoin.d.ts",
+      "require": "./lib-commonjs/PersonaCoin.js"
+    },
+    "./lib-commonjs/PersonaPresence": {
+      "types": "./lib-commonjs/PersonaPresence.d.ts",
+      "require": "./lib-commonjs/PersonaPresence.js"
+    },
+    "./lib-commonjs/Pickers": {
+      "types": "./lib-commonjs/Pickers.d.ts",
+      "require": "./lib-commonjs/Pickers.js"
+    },
+    "./lib-commonjs/Pivot": {
+      "types": "./lib-commonjs/Pivot.d.ts",
+      "require": "./lib-commonjs/Pivot.js"
+    },
+    "./lib-commonjs/Popup": {
+      "types": "./lib-commonjs/Popup.d.ts",
+      "require": "./lib-commonjs/Popup.js"
+    },
+    "./lib-commonjs/Positioning": {
+      "types": "./lib-commonjs/Positioning.d.ts",
+      "require": "./lib-commonjs/Positioning.js"
+    },
+    "./lib-commonjs/PositioningContainer": {
+      "types": "./lib-commonjs/PositioningContainer.d.ts",
+      "require": "./lib-commonjs/PositioningContainer.js"
+    },
+    "./lib-commonjs/ProgressIndicator": {
+      "types": "./lib-commonjs/ProgressIndicator.d.ts",
+      "require": "./lib-commonjs/ProgressIndicator.js"
+    },
+    "./lib-commonjs/Rating": {
+      "types": "./lib-commonjs/Rating.d.ts",
+      "require": "./lib-commonjs/Rating.js"
+    },
+    "./lib-commonjs/ResizeGroup": {
+      "types": "./lib-commonjs/ResizeGroup.d.ts",
+      "require": "./lib-commonjs/ResizeGroup.js"
+    },
+    "./lib-commonjs/ResponsiveMode": {
+      "types": "./lib-commonjs/ResponsiveMode.d.ts",
+      "require": "./lib-commonjs/ResponsiveMode.js"
+    },
+    "./lib-commonjs/ScrollablePane": {
+      "types": "./lib-commonjs/ScrollablePane.d.ts",
+      "require": "./lib-commonjs/ScrollablePane.js"
+    },
+    "./lib-commonjs/SearchBox": {
+      "types": "./lib-commonjs/SearchBox.d.ts",
+      "require": "./lib-commonjs/SearchBox.js"
+    },
+    "./lib-commonjs/SelectableOption": {
+      "types": "./lib-commonjs/SelectableOption.d.ts",
+      "require": "./lib-commonjs/SelectableOption.js"
+    },
+    "./lib-commonjs/SelectedItemsList": {
+      "types": "./lib-commonjs/SelectedItemsList.d.ts",
+      "require": "./lib-commonjs/SelectedItemsList.js"
+    },
+    "./lib-commonjs/Selection": {
+      "types": "./lib-commonjs/Selection.d.ts",
+      "require": "./lib-commonjs/Selection.js"
+    },
+    "./lib-commonjs/Separator": {
+      "types": "./lib-commonjs/Separator.d.ts",
+      "require": "./lib-commonjs/Separator.js"
+    },
+    "./lib-commonjs/Shimmer": {
+      "types": "./lib-commonjs/Shimmer.d.ts",
+      "require": "./lib-commonjs/Shimmer.js"
+    },
+    "./lib-commonjs/ShimmeredDetailsList": {
+      "types": "./lib-commonjs/ShimmeredDetailsList.d.ts",
+      "require": "./lib-commonjs/ShimmeredDetailsList.js"
+    },
+    "./lib-commonjs/Slider": {
+      "types": "./lib-commonjs/Slider.d.ts",
```

Comment:
types for cjs or esm are no different. wondering if this is really needed as its lot of duplication

Reply:
If a require resolves to lib-commonjs, it will look here for types. We *could* map this to ./lib/* but I think the entry still needs to be there.

Reply:
> We could map this to ./lib/* but I think the entry still needs to be there.

that would be great! can you check if it is needed ? ty

Reply:
The d.ts files exist in both cjs and esm folders.

https://unpkg.com/browse/@fluentui/react@8.94.4/lib-commonjs/
https://unpkg.com/browse/@fluentui/react@8.94.4/lib/

Going to leave as is for now unless this changes and we want to reduce the package size. This would require more build alterations to emit typings into a common folder.